docs/tutorials/network/espnow.mdx

```diff
@@ -0,0 +1,257 @@
+# ESP-NOW
+ESP-NOW is a proprietary protocol developed by Espressif for
+low-power, connectionless communication between ESP32 devices.
+
+In this tutorial we will set up two ESP32 where one will send
+messages, and the other will receive them.
+
+## Prerequisites
+We assume that you have set up your development environment as described
+in [the IDE tutorial](../../setup/ide).
+
+We also assume that you have flashed your device with Jaguar and that
+you are familiar with running Toit programs on it.
+If not, have a look at the [Hello world](../../setup/firstprogram) tutorial.
+
+<Note>
+
+When working with two devices, you will need to tell Jaguar which
+USB port to use.
+
+When using `jag monitor` you have to use the `--port` option to specify
+the port the ESP32 device is connected to. For example:
+
+``` shell
+jag monitor --port COM3          # Windows.
+jag monitor --port /dev/ttyUSB0  # Linux and macOS.
+```
+
+</Note>
+
+## Introduction
+
+ESP-NOW can be seen as a replacement for Wifi, where devices don't
+need to connect to an access point, but can communicate directly with
+each other. Its range is typically better than Wifi, but it has
+lower data rates.
+
+Devices can either communicate by broadcasting messages to all
+devices in range, or by sending messages to specific devices using
+their MAC address. In both cases, a key can be used to encrypt the
+messages.
+
+## Limitations
+
+Messages are at most 1470 bytes long. For broadcast messages there
+is no guarantee that messages are correctly delivered. For direct
+messages, the implementation ensures that the PHY layer has
+received the message, but there is no guarantee that the message
+is processed by the receiving device.
+
+ESP-NOW only supports 20 peers, although one of them can be the
```

Comment:
Is peers correct here? How can a peer be a broadcast address?

Reply:
done.